src/common/checksum.c

```diff
@@ -90,6 +90,28 @@ daos_csummer_init_with_props(struct daos_csummer **obj, daos_prop_t *props)
 					   daos_cont_prop2chunksize(props),
 					   daos_cont_prop2serververify(props));
 }
+/*
```

Comment:
my guess is the linter actually wants a blank line here

Reply:
I will add blank lines if I need to push a new patch, but if I get approvals I would like to land this without resubmitting to CI and will add those lines in another PR.